unipipe/main.ts

```diff
@@ -19,4 +19,7 @@ registerTransformCmd(program);
 registerUpdateCmd(program);
 registerGenerateCmd(program);
 
-await program.parse(Deno.args);
+if (Deno.args.length > 0)
+  await program.parse(Deno.args);
+else
+  program.showHelp()
```

Comment:
d: I noticed that this issue also exists with commands that have subcommands like `unipipe generate` -> do you also have an idea how to fix those?

Reply:
afterwards i have also figured out that and i believe it impacts the user experience a little bit when you see an empty response after you execute something. i will look at them as well before merging, so maybe i can easily fix them all at the same time